bin/cronjob

```diff
@@ -11,14 +11,14 @@
 #   or returned output.
 #
 # ERROR (not currently differentiated): A non-0 exit code could be elevated to mean error and
-#   result in a red HipChat message and/or channel notification.
+#   result in a red Slack message and/or channel notification.
 #
 # CRONJOBs are defined the cdo-apps::crontab Chef recipe: `chef/cookbooks/cdo-apps/recipes/crontab.rb`.
 #
 
 require_relative '../deployment.rb'
 require 'mail'
-require 'cdo/hip_chat'
+require 'cdo/chat_client'
```

Comment:
Though this appears unused, I'm afraid of transitive dependency, so am not removing.